test/text-field.html

```diff
@@ -216,6 +216,24 @@
             });
           });
         }
+
+        describe(`autoselect ${condition}`, function() {
+          it('default value of autoselect should be false', function() {
+            expect(textField.autoselect).to.be.false;
+          });
```

Comment:
nit: please add newline between tests (after each test except last one) as in the blocks above to keep formatting consistent

Reply:
Done.